### PR TITLE
Set up 'Getting started' page

### LINKED ISSUE #19: Replace 'Installation' with a new 'Getting started'
Describe how to use:

- playground
- runner

### PR DESCRIPTION
**Pull request by fsteeg**
Will resolve #19.

### CODE REVIEW

File: getting-started.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<p>To use Metafacture as a command-line tool, download the latest metafix-runner from our <a href="https://github.com/metafacture/metafacture-fix/releases">releases page</a>. Extract the downloaded archive and change into the newly created directory (e.g. <code>cd metafacture-runner-0.4.0</code>). Run a Flux workflow with:</p>
+<p><code>$ ./bin/metafix-runner /path/to/your.flux</code> on Unix/Linux/Mac or <code>$ ./bin/metafix-runner.bat /path/to/your.flux</code> on Windows.</p>
+<p>To get started, you can export a workflow from the Playground (last button, “Export Workflow”).</p>
+<p>To set up IDE support for editing your Flux and Fix files, see <a href="/ide-extensions/index.html">the IDE extensions page</a>.</p>
+<h2 id="next-steps">Next steps</h2>
+<p>For a complete introduction to Metafacture in German, check out the latest iteration of <a href="https://slides.lobid.org/2022-12-metafacture-workshop/#/">our Metafacture Workshop</a>. For different use cases, e.g. using Metafacture as a library, using the Morph language, and more, see <a href="https://github.com/metafacture/metafacture-documentation">our documentation collection</a>.</p>
+</body>
```
Comment (TobiasNx) [inline getting-started.html:56]: Should we link a guide for the Java Integration too?

File: getting-started.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Getting started"
+header-includes:
+    <link rel="Metafacture Icon" type="image/x-icon" href="/img/metafacture-icon.png">
+---
+
+## Playground
+
+The easiest way to get started with Metafacture is the Playground. Take a look at the [first example](https://metafacture.org/playground/?flux=PG_DATA%0A%7Cas-lines%0A%7Cdecode-formeta%0A%7Cfix%0A%7Cencode-xml%28rootTag%3D%22collection%22%29%0A%7Cprint%0A%3B&fix=move_field%28_id%2C+id%29%0Amove_field%28a%2C+title%29%0Apaste%28author%2C+b.v%2C+b.n%2C+%27~aus%27%2C+c%29%0Aretain%28id%2C+title%2C+author%29&data=1%7Ba%3A+Faust%2C+b+%7Bn%3A+Goethe%2C+v%3A+JW%7D%2C+c%3A+Weimar%7D%0A2%7Ba%3A+R%C3%A4uber%2C+b+%7Bn%3A+Schiller%2C+v%3A+F%7D%2C+c%3A+Weimar%7D&active-editor=fix) and run it by pressing the "Process" button. Check out the other examples (first button, "Load Examples") for different input sources, transformations, and output formats.
```
[the remark this answers is not in the document]
Comment (fsteeg) [reply getting-started.md:9]: Sounds good!

**Comment by TobiasNx** [comment]
@fsteeg process fails

**Comment by fsteeg** [comment]
Updated the action to use the `ubuntu-latest` runner in https://github.com/metafacture/metafacture.github.io/pull/20/commits/e95dde9382cd277a72cbd6e2da7599a102f0a380 and added a missing update in https://github.com/metafacture/metafacture.github.io/pull/20/commits/75b82bfd1183093905ab1a9bddde30ee690dd438, now it looks good, HTML was committed by the GitHub Action in https://github.com/metafacture/metafacture.github.io/pull/20/commits/09a88de6a06602a3726704e9e5fddb6fec3e7382.

**Review by katauber — APPROVED** [review]

**Pull request merged**